articles/libraries/numerics/numerics.md

```diff
@@ -0,0 +1,148 @@
+    - **`ComputeReciprocalFxP`**: Compute 1/x for a quantum fixed-point number `x`
+    - **`MeasureFxP`**: Measure a fixed-point number and return a Double
+
+For more information and detailed documentation for each of these operations, see the Q# library reference docs at [docs.microsoft.com](https://docs.microsoft.com/en-us/quantum)
+
+## Sample: Integer addition
+
+As a basic example, consider the operation
+$$
+\ket x\ket y\mapsto \ket x\ket{x+y}
+$$
+that is, an operation that takes an n-qubit integer $x$ and an n- or (n+1)-qubit
+register $y$ as input, the latter of which it maps to the sum $(x+y)$. Note that the
+sum is computed modulo $2^n$ if $y$ is stored in an $n$-bit register.
+
+Using the QDK, this operation can be applied as follows
+```qsharp
+operation MyAdditionTest (xInt : Int, yInt : Int, n : Int) : Unit
+{
+    using ((xQubits, yQubits) = (Qubit[n], Qubit[n]))
+    {
+        x = LittleEndian(xQubits); // define bit order
+        y = LittleEndian(yQubits);
+        
+        ApplyXorInPlace(xInt, x); // initialize values
+        ApplyXorInPlace(yInt, y);
+        
+        AddI(x, y); // perform addition x+y into y
+        
+        // ... (use the result)
+    }
+}
+```
+
+## Sample: Evaluating smooth functions
+
+To evaluate smooth functions such as $\sin(x)$ on a quantum computer, where $x$ is a quantum `FixedPoint` number,
+the QDK Numerics Library provides the operations `EvaluatePolynomialFxP` and `Evaluate[Even/Odd]PolynomialFxP`.
+
+The first, `EvaluatePolynomialFxP`, allows to evaluate a polynomial of the form
+$$
+P(x) = a_0 + a_1x + a_2x^2 + \cdots + a_dx^d,
+$$
+where $d$ denotes the *degree*. To do so, all that is needed are the polynomial coefficients `[a_0,..., a_d]` (of type `Double[]`),
+the input `x : FixedPoint` and the output `y : FixedPoint` (initially zero):
+```qsharp
+EvaluatePolynomialFxP([1.0, 2.0], xFxP, yFxP);
+```
+The result, $P(x)=1+2x$, will be stored in `yFxP`.
+
+The second, `EvaluateEvenPolynomialFxP`, and the third, `EvaluateOddPolynomialFxP`, are specializations
+for the cases of even and odd functions, respectively. That is, for an even/odd function $f(x)$ and
+$$
+P_{even}(x)=a_0 + a_1 x^2 + a_2 x^4 + \cdots + a_d x^{2d},
+$$
+$f(x)$ is approximated well by $P_{even}(x)$ or $P_{odd}(x) := x\cdot P_{even}(x)$, respectively.
+In Q#, these two cases can be handled as follows:
+```qsharp
+EvaluateEvenPolynomialFxP([1.0, 2.0], xFxP, yFxP);
+```
+which evaluates $P_{even}(x) = 1 + 2x^2$, and
+```qsharp
+EvaluateOddPolynomialFxP([1.0, 2.0], xFxP, yFxP);
+```
+which evaluates $P_{odd}(x) = x + 2x^3$.
+
+## More samples
+
+You can find more samples in the [main samples repository](https://github.com/Microsoft/Quantum).
+
+To get started, clone the repo and open the `Numerics` subfolder:
+
+```bash
+git clone https://github.com/Microsoft/Quantum.git
+cd Quantum/Numerics
+```
+
+Then, `cd` into one of the sample folders and run the sample via
+
+```bash
+cd [SampleFolder]
```

Comment:
Readers still learning how to navigate the command line may think that this is literal syntax; could we perhaps do something like give a concrete example, and say "for example" at L143?